### PR TITLE
feat: add raft store fail metrics

### PR DESCRIPTION
**Pull request by lichuang**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

feat: add raft store fail metrics

Fixes #issue https://github.com/datafuselabs/databend/issues/6909

### CODE REVIEW

**Comment by Xuanwo** [comment]
image

The page is broken: https://databend-git-fork-lichuang-metasrvmetrics-databend.vercel.app/doc/manage/metasrv/metasrv-metrics

File: docs/doc/50-manage/00-metasrv/50-metasrv-metrics.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ These metrics describe the network status of raft nodes in the `metasrv`. All th
 | snapshot_sent_seconds   | Total latency distributions of snapshot sends.    | to(node id)                       | HistogramVec  |
 | snapshot_recv_success   | Total number of successful receive snapshot.      | from(remote address)              | IntCounterVec |
 | snapshot_recv_failures  | Total number of snapshot receive failures.        | from(remote address)              | IntCounterVec |
+
```
Comment (Xuanwo) [inline docs/doc/50-manage/00-metasrv/50-metasrv-metrics.md:63]: Why do we add a new line here?

File: docs/doc/50-manage/00-metasrv/50-metasrv-metrics.md [hunk 2]
```diff
@@ -75,10 +76,21 @@ These metrics describe the network status of raft nodes in the `metasrv`. All th
...
+| raft_store_write_failed            | Total number of raft store write failures.    | func(function name) | CounterVec      |
+| raft_store_read_failed            | Total number of raft store read failures.    | func(function name) | CounterVec      |
+
+`raft_store_write_failed` and `raft_store_read_failed` indicate the total number of raft store write and read failures.
+
 ### Meta Network
 
 These metrics describe the network status of meta service in the `metasrv`. All these metrics are prefixed with `metasrv_meta_network_`.
-
+i
```
Comment (Xuanwo) [inline docs/doc/50-manage/00-metasrv/50-metasrv-metrics.md:93]: typo?

**Review by drmingdrmer — APPROVED** [review]

**Comment by Xuanwo** [comment]
Still broken:

image

For previewing website locally: `cd website && yarn start`

**Review by Xuanwo — APPROVED** [review]

**Pull request merged**